docs/guides/server/fips.adoc

```diff
@@ -35,7 +35,13 @@ BouncyCastle FIPS can be downloaded from the https://www.bouncycastle.org/fips-j
 
 You can create either `pkcs12` or `bcfks` keystore to be used for the Keycloak server SSL. The `pkcs12` works well in BCFIPS non-approved mode.
 
-PKCS12 keystore can be generated with OpenJDK 17 Java on RHEL 9 in the standard way.
+PKCS12 keystore can be generated with OpenJDK 17 Java on RHEL 9 in the standard way. Make sure that the system is in FIPS mode, you can check it with the following command:
+
+```bash
+fips-mode-setup --check
```

Comment:
Isn't  `fips-mode-setup --enabled` the command that actually changes the Java security files?

Reply:
Correct, originally there was `--enabled` param, then I realized that I meant `--check` param for checking. And I forgot to change the following sentence.

Let me fix it quickly.